src/MessagePack.UnityClient/Assets/Scripts/Tests/ShareTests/MessagePackSecurityTests.cs

```diff
@@ -5,11 +5,17 @@
 using System.Collections.Generic;
 using System.Runtime.Serialization;
 using MessagePack;
+using MessagePack.Tests;
 using Xunit;
 using Xunit.Abstractions;
 
 public class MessagePackSecurityTests
 {
+    public MessagePackSecurityTests()
+    {
+        Logger = new NullTestOutputHelper();
```

Comment:
Will Xunit still choose the other overload of the constructor, or will this new default constructor effectively break logging in xunit?

Reply:
Unity's test for IL2CPP (Cysharp/RuntimeUnitTestToolkit) only supports parameter-less constructors; it shares the Unit Test between xUnit and Unity so require to support both.

Reply:
I understand why we need the default constructor. What I'm concerned about is whether xunit (the real one, outside of Unity) will still invoke the original one that takes an `ITestOutputHelper` parameter. If it doesn't, and chooses the default constructor instead, we'll lose logging when running tests outside Unity.

Reply:
Yes, xUnit only allows single constructor. I've enclosed parameter-less constructor with `#if UNITY_2018_3_OR_NEWER`.